community/logging/src/main/java/org/neo4j/logging/RotatingFileOutputStreamSupplier.java

```diff
@@ -292,26 +294,34 @@ private void rotate()
                         if ( !closed.get() && outRef.equals( nullStream ) )
                         {
                             outRef = openOutputFile();
-                            rotationListener.outputFileCreated( streamWrapper );
+                            rotationListener.outputFileCreated( bufferingOutputStream );
                         }
                     }
                     catch ( IOException e )
                     {
                         System.err.println( "Failed to open log file after log rotation: " + e.getMessage() );
-                        rotationListener.rotationError( e, streamWrapper );
+                        rotationListener.rotationError( e, bufferingOutputStream );
```

Comment:
That print was put there in case no log file could be opened, in which case the log will be sent to the `nullStream`. The user would have no indication that something is wrong.